Improve output of validation

rdohms opened the pull request:
Made the output more redable by listing the errors, being less verbose
and using colors to signal success/failure.

rdohms commented:
@lcobucci exit codes validated, they are now correct
```
circleci@2b6241588b50:/project$ echo `python validator.py test.yaml`
 [PASS] No errors found
circleci@2b6241588b50:/project$ echo $?
0
circleci@2b6241588b50:/project$ python validator.py test.yaml

Error #1 in [paths:/feedback:post:responses:202]:
    {'description': 'Feedback Item succesfully queued for processing', 'headers': {'Location': {'description': 'Location of Feedback Item resource', 'schemax': {'type': 'string', 'format': 'url'}, 'example': 'http://api.com/83eb4be7-138e-41d5-ae18-4a42fe3cd911'}}} is not valid under any of the given schemas
    {'description': 'Feedback Item succesfully queued for processing', 'headers': {'Location': {'description': 'Location of Feedback Item resource', 'schemax': {'type': 'string', 'format': 'url'}, 'example': 'http://api.com/83eb4be7-138e-41d5-ae18-4a42fe3cd911'}}}

 [FAIL] 1 errors found
circleci@2b6241588b50:/project$ echo $?
1
```

lcobucci commented on validator.py:
```diff
@@ -1,23 +1,58 @@
 
 def main(argv):
-    if (len(argv) != 1):
+    if len(argv) == 0:
```
Not sure if we should do it but let's get this merged and we discuss afterwards (it's not really related to what we're doing now).

rdohms replied:
This was done so I could actually run it in pycharm, since it appends `<host> <ip>` o something like that to all calls

lcobucci replied:
Ok, no worries. But maybe, you know, separate commits =)

lcobucci commented on validator.py:
```diff
@@ -1,23 +1,58 @@
+
+        handler = UrlHandler('file')
+        spec = handler('file://' + path.join(getcwd(), file_name))
+        errors_iterator = openapi_v3_spec_validator.iter_errors(spec)
```
Was trying this out and it fails if we have references to other files because the spec URL is not being sent (like in https://github.com/p1c2u/openapi-spec-validator/blob/7d3bc1d7ff13b6416b3311999aa42740efc5bfb5/openapi_spec_validator/shortcuts.py#L16)

rdohms replied:
Did it work with the original code? Otherwise I would handle that in a separate issue

lcobucci replied:
Yeap, it did, mostly because I was using the provided wrapper (that encapsulates the `UrlHandler`)

lcobucci commented:
@rdohms I've merged in #3 so that we prevent ourselves from violating the sacred code standards. Please rebase this to sync changes ;)

rdohms committed:
Only stop execution if no argument has been given

So that the script can be executed by IDE's, since they add extra
arguments to the call.

rdohms committed:
Improve output of validation

Made the output more redable by listing the errors, being less verbose
and using colors to signal success/failure.
